### PR TITLE
feat(gf/gen/pbentity): add a `TablesEx` configuration to exclude the specified table names

### PR DESCRIPTION
**Pull request by wwwfeng**
related issue #3955 

image

### CODE REVIEW

**Comment by gqcn** [comment]
@wwwfeng The CI failed, you may execute `git pull https://github.com/gogf/gf master` to update your repo.

**Comment by wwwfeng** [comment]
> @wwwfeng The CI failed, you may execute `git pull https://github.com/gogf/gf master` to update your repo.

image
Strange, this is already the latest master branch.

**Comment by gqcn** [comment]
> > @wwwfeng The CI failed, you may execute `git pull https://github.com/gogf/gf master` to update your repo.
> 
> image Strange, this is already the latest master branch.

OK, let's ignore the golint error in CI.

File: cmd/gf/internal/cmd/genpbentity/genpbentity.go [hunk 1]
```diff
@@ -43,6 +44,7 @@ type (
 		Prefix            string `name:"prefix"            short:"f"  brief:"{CGenPbEntityBriefPrefix}"`
 		RemovePrefix      string `name:"removePrefix"      short:"r"  brief:"{CGenPbEntityBriefRemovePrefix}"`
 		RemoveFieldPrefix string `name:"removeFieldPrefix" short:"rf" brief:"{CGenPbEntityBriefRemoveFieldPrefix}"`
+		TableExclude      string `name:"tableExclude"      short:"te" brief:"{CGenPbEntityBriefTableExclude}"`
```
Comment (gqcn) [inline cmd/gf/internal/cmd/genpbentity/genpbentity.go:47]: Please use `TableEx` as the field name, to be the same as command `gen dao`: https://github.com/gogf/gf/blob/817ac36ce24af81b05e663a9d4f90e479b76d52a/cmd/gf/internal/cmd/gendao/gendao.go#L188

**Review by gqcn — CHANGES_REQUESTED** [review]

File: cmd/gf/internal/cmd/genpbentity/genpbentity.go (hunk 1 above)
Comment (wwwfeng) [reply cmd/gf/internal/cmd/genpbentity/genpbentity.go:47]: done

**Review by gqcn — APPROVED** [review]

**Pull request merged**